Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -484,6 +495,20 @@ class CredentialsManagerSpec: QuickSpec {
                         }
                     }
                 }
+
+                it("should include custom headers") {
+                    stub(condition: hasHeader("foo", value: "bar")) {
+                        _ in return authResponse(accessToken: NewAccessToken, idToken: NewIdToken, refreshToken: NewRefreshToken, expiresIn: ExpiresIn)
+                    }
+                    credentials = Credentials(accessToken: AccessToken, tokenType: TokenType, idToken: IdToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: -ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: Timeout) { done in
+                        credentialsManager.credentials(headers: ["foo": "bar"]) { result in
+                            expect(result).to(haveCredentials(NewAccessToken, NewIdToken, NewRefreshToken))
```

Comment:
Same as above, this probably should be only asserting that the result was successful.